components/tidb_query_datatype/src/codec/data_type/chunked_vec_set.rs

```diff
@@ -75,7 +74,7 @@ impl ChunkedVec<Set> for ChunkedVecSet {
     }
 
     fn capacity(&self) -> usize {
-        self.bitmap.capacity().max(self.value.capacity())
+        self.bitmap.capacity().min(self.value.capacity())
```

Comment:
Why changed to `min`?

Reply:
I encountered some failing tests because the capacity of `bit_vec::BitVec` behaves a bit differently than the custom `BitVec` - it is always a multiple of 32.

My thinking is that in this instance for example the capacity should be the minimum of the two, because to add a new element without an allocation, you need to have an empty slot in *both* data structures. My reasoning is similar in `chunked_vec_enum.rs`